lib/wasi/src/os/console/txt/about_deploy.md

```diff
@@ -0,0 +1,14 @@
+# wasmer.sh
```

Comment:
I don't understand why this file is here.

Reply:
Removed.